ops/_private/harness.py

```diff
@@ -3743,6 +3805,41 @@ def get_checks(
             if (level is None or level == info.level) and (names is None or info.name in names)
         ]
 
+    def start_checks(self, names: List[str]):
+        self._check_connection()
+        for name in names:
+            if name not in self._check_infos:
+                raise self._api_error(404, f'cannot find check with name "{name}"')
+            info = self._check_infos[name]
+            if not info.change_id:
+                info.status = pebble.CheckStatus.UP
+                info.failures = 0
+                change = pebble.Change(
```

Comment:
Perhaps factor this out (occurs 3 times) into `change = self._perform_check_change()` or `change = self._new_change(pebble.ChangeKind.PERFORM_CHECK` or something?